Command failed: git ls-tree -r --full-tree

**🐞 Describe the bug**

The actions fails with `Error: Command failed: git ls-tree -r --full-tree `

**📚 To Reproduce**

Please see my config here: 

- Action: https://github.com/Nef10/swiftlint-config/blob/main/.github/workflows/sync.yml
- Input: https://github.com/Nef10/swiftlint-config/blob/main/.github/sync.yml

**💡 Expected behavior**

Should sync correctly. 

**📄 Logs**

The run is here: https://github.com/Nef10/swiftlint-config/runs/6332482552?check_suite_focus=true

<details>
  <summary>Debug Log</summary>

```
2022-05-07T05:34:10.0592535Z ##[debug]Loading env
2022-05-07T05:34:10.0599012Z ##[group]Run BetaHuhn/repo-file-sync-action@v1
2022-05-07T05:34:10.0599458Z with:
2022-05-07T05:34:10.0600044Z   GH_INSTALLATION_TOKEN: ***
2022-05-07T05:34:10.0600547Z   GIT_EMAIL: 89714990+file-sync-app[bot]@users.noreply.github.com
2022-05-07T05:34:10.0601085Z   GIT_USERNAME: file-sync-app[bot]
2022-05-07T05:34:10.0601517Z   ORIGINAL_MESSAGE: true
2022-05-07T05:34:10.0601931Z   COMMIT_EACH_FILE: false
2022-05-07T05:34:10.0602326Z   SKIP_PR: true
2022-05-07T05:34:10.0602709Z ##[endgroup]
2022-05-07T05:34:10.2341759Z ::add-mask::***
2022-05-07T05:34:10.2347320Z ##[debug]{
2022-05-07T05:34:10.2348036Z ##[debug]  "GITHUB_TOKEN": "***",
2022-05-07T05:34:10.2348680Z ##[debug]  "IS_INSTALLATION_TOKEN": true,
2022-05-07T05:34:10.2349236Z ##[debug]  "GIT_EMAIL": "89714990+file-sync-app[bot]@users.noreply.github.com",
2022-05-07T05:34:10.2349812Z ##[debug]  "GIT_USERNAME": "file-sync-app[bot]",
2022-05-07T05:34:10.2350290Z ##[debug]  "CONFIG_PATH": ".github/sync.yml",
2022-05-07T05:34:10.2350721Z ##[debug]  "COMMIT_BODY": "",
2022-05-07T05:34:10.2351162Z ##[debug]  "COMMIT_PREFIX": "🔄",
2022-05-07T05:34:10.2351583Z ##[debug]  "COMMIT_EACH_FILE": false,
2022-05-07T05:34:10.2352005Z ##[debug]  "PR_LABELS": [
2022-05-07T05:34:10.2352387Z ##[debug]    "sync"
2022-05-07T05:34:10.2352744Z ##[debug]  ],
2022-05-07T05:34:10.2353107Z ##[debug]  "PR_BODY": "",
2022-05-07T05:34:10.2353517Z ##[debug]  "TMP_DIR": "tmp-1651901650229",
2022-05-07T05:34:10.2353941Z ##[debug]  "DRY_RUN": false,
2022-05-07T05:34:10.2354339Z ##[debug]  "SKIP_CLEANUP": false,
2022-05-07T05:34:10.2354763Z ##[debug]  "OVERWRITE_EXISTING_PR": true,
2022-05-07T05:34:10.2355255Z ##[debug]  "GITHUB_REPOSITORY": "Nef10/swiftlint-config",
2022-05-07T05:34:10.2355717Z ##[debug]  "SKIP_PR": true,
2022-05-07T05:34:10.2356115Z ##[debug]  "ORIGINAL_MESSAGE": true,
2022-05-07T05:34:10.2356849Z ##[debug]  "COMMIT_AS_PR_TITLE": false,
2022-05-07T05:34:10.2357326Z ##[debug]  "BRANCH_PREFIX": "repo-sync/SOURCE_REPO_NAME",
2022-05-07T05:34:10.2357785Z ##[debug]  "FORK": false
2022-05-07T05:34:10.2358343Z ##[debug]}
2022-05-07T05:34:10.2466685Z Repository Info
2022-05-07T05:34:10.2491513Z Slug		: common-swift-package
2022-05-07T05:34:10.2492197Z Owner		: Nef10
2022-05-07T05:34:10.2493240Z Https Url	: https://github.com/Nef10/common-swift-package
2022-05-07T05:34:10.2494526Z Branch		: default
2022-05-07T05:34:10.2494887Z 	
2022-05-07T05:34:10.2495904Z ##[debug]Cloning github.com/Nef10/common-swift-package into tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.2497824Z ##[debug]EXEC: "git clone --depth 1  ***github.com/Nef10/common-swift-package.git tmp-1651901650229/github.com/Nef10/common-swift-package@default" IN undefined
2022-05-07T05:34:10.4376154Z ##[debug]Setting git user to email: 89714990+file-sync-app[bot]@users.noreply.github.com, username: file-sync-app[bot]
2022-05-07T05:34:10.4377739Z ##[debug]EXEC: "git config --local user.name "file-sync-app[bot]" && git config --local user.email "89714990+file-sync-app[bot]@users.noreply.github.com"" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.4452994Z ##[debug]EXEC: "git rev-parse --abbrev-ref HEAD" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.4509802Z ##[debug]EXEC: "git rev-parse HEAD" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.4537622Z Locally syncing file(s) between source and target repository
2022-05-07T05:34:10.4549900Z ##[debug]CP: .swiftlint.yml TO tmp-1651901650229/github.com/Nef10/common-swift-package@default/.swiftlint.yml
2022-05-07T05:34:10.4573152Z ##[debug]EXEC: "git add -f ".swiftlint.yml"" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.4617424Z ##[debug]EXEC: "git status --porcelain" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.4664237Z ##[debug]Creating commit for remaining files
2022-05-07T05:34:10.7047389Z ##[debug]EXEC: "git diff HEAD .swiftlint.yml" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7110507Z ##[debug]EXEC: "git commit -m 'Update SwiftLint rules for 0.47.1'" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7189949Z Pushing changes to target repository
2022-05-07T05:34:10.7191456Z ##[debug]EXEC: "git log --format=%H --reverse main..HEAD" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7244940Z ##[debug]EXEC: "git log -1 --format=%B " IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7268692Z ##[debug]EXEC: "git ls-tree -r --full-tree " IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7283222Z ##[debug]Creating missing blobs on GitHub
2022-05-07T05:34:10.7284569Z ##[debug]EXEC: "git ls-tree -r --full-tree ~1" IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7307442Z ##[debug]EXEC: "git ls-tree -r --full-tree " IN tmp-1651901650229/github.com/Nef10/common-swift-package@default
2022-05-07T05:34:10.7353249Z ##[error]Command failed: git ls-tree -r --full-tree 
usage: git ls-tree [<options>] <tree-ish> [<path>...]

    -d                    only show trees
    -r                    recurse into subtrees
    -t                    show trees when recursing
    -z                    terminate entries with NUL byte
    -l, --long            include object size
    --name-only           list only filenames
    --name-status         list only filenames
    --object-only         list only objects
    --full-name           use full path names
    --full-tree           list entire tree; not just current directory (implies --full-name)
    --format <format>     format to use for the output
    --abbrev[=<n>]        use <n> digits to display object names


2022-05-07T05:34:10.7363115Z ##[debug]{"killed":false,"code":129,"signal":null,"cmd":"git ls-tree -r --full-tree "}
```

</details>

**📋 Additional context**

Last ran synced correctly with the same config on Jan 22.

--- a/src/git.js
+++ b/src/git.js
@@ -16,6 +16,7 @@ const {
 	COMMIT_PREFIX,
 	GITHUB_REPOSITORY,
 	OVERWRITE_EXISTING_PR,
+	SKIP_PR,
 	PR_BODY,
 	BRANCH_PREFIX,
 	FORK
@@ -288,7 +289,7 @@ class Git {
 	// Gets the commit list in chronological order
 	async getCommitsToPush() {
 		const output = await execCmd(
-			`git log --format=%H --reverse ${ this.baseBranch }..HEAD`,
+			`git log --format=%H --reverse ${ SKIP_PR === false ? `` : `origin/` }${ this.baseBranch }..HEAD`,
 			this.workingDir
 		)
 
@@ -323,30 +324,32 @@ class Git {
 	async createGithubVerifiedCommits() {
 		const commitsData = await this.getCommitsDataToPush()
 
-		// Creates the PR branch if doesn't exists
-		try {
-			await this.github.git.createRef({
-				owner: this.repo.user,
-				repo: this.repo.name,
-				sha: this.lastCommitSha,
-				ref: 'refs/heads/' + this.prBranch
-			})
+		if (SKIP_PR === false) {
+			// Creates the PR branch if doesn't exists
+			try {
+				await this.github.git.createRef({
+					owner: this.repo.user,
+					repo: this.repo.name,
+					sha: this.lastCommitSha,
+					ref: 'refs/heads/' + this.prBranch
+				})
 
-			core.debug(`Created new branch ${ this.prBranch }`)
-		} catch (error) {
-			// If the branch exists ignores the error
-			if (error.message !== 'Reference already exists') throw error
+				core.debug(`Created new branch ${ this.prBranch }`)
+			} catch (error) {
+				// If the branch exists ignores the error
+				if (error.message !== 'Reference already exists') throw error
+			}
 		}
 
 		for (const commitData of commitsData) {
 			await this.createGithubTreeAndCommit(commitData.tree, commitData.commitMessage)
 		}
 
-		core.debug(`Updating branch ${ this.prBranch } ref`)
+		core.debug(`Updating branch ${ SKIP_PR === false ? this.prBranch : this.baseBranch } ref`)
 		await this.github.git.updateRef({
 			owner: this.repo.user,
 			repo: this.repo.name,
-			ref: `heads/${ this.prBranch }`,
+			ref: `heads/${ SKIP_PR === false ? this.prBranch : this.baseBranch }`,
 			sha: this.lastCommitSha,
 			force: true
 		})

